src/main/java/pages/header/LogUserInfo.java

```diff
@@ -46,4 +46,9 @@ public boolean findSignOut() {
     public void clickUserAccount() {
         userAccount.click();
     }
+    
+    public String getUserAccount () {
```

Comment:
Maybe better named it "getUserAccountText" or smth like that, because it looks like simple getter

Reply:
ok, in next stage it will be corrected